Kentico.Kontent.Management/Models/Subscription/SubscriptionUserEnvironmentModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Kentico.Kontent.Management.Models.Subscription
+{
+    /// <summary>
+    /// Represents projects to which the user has been invited.
+    /// </summary>
+    public sealed class SubscriptionUserEnvironmentModel
```

Comment:
Let's keep the hierarchy naming from https://kontent.ai/learn/reference/management-api-v2/#section/Subscription-user-object

```suggestion
    public sealed class SubscriptionUserProjectEnvironmentModel
```

Check also models for [Subscription project object](https://kontent.ai/learn/reference/management-api-v2/#section/Subscription-project-object)